Makefile

```diff
@@ -33,7 +34,7 @@ submodule-init:
 .PHONY: integration
 integration: submodule-init setup-ckb-test ## Run integration tests in "test" dir.
 	cargo build
-	cd test && RUST_BACKTRACE=1 cargo run -- --bin ../target/debug/ckb ${CKB_TEST_ARGS}
+	cd test && RUST_BACKTRACE=1 RUST_LOG=ckb-network=error cargo run -- --bin ../target/debug/ckb ${CKB_TEST_ARGS}
```

Comment:
Please remove this `RUST_LOG`, so, we can set it via `export` environment variable.